### PR TITLE
Adds Database connection

### PR DESCRIPTION
**Pull request by rahulc0dy**
## Explanation

- Added connection to database
- Database Health check endpoint added
- Health check routes merged to a single route with base  `/health-check`

## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR does not contain any unnecessary code changes from code editor.
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers.

### CODE REVIEW

File: src/db/database.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const sql = neon(process.env.DATABASE_URL);
+
+/* request handler structure */
+// const requestHandler = async (req, res) => {
+//   const result = await sql`SELECT version()`;
+//   const { version } = result[0];
+//   res.writeHead(200, { "Content-Type": "text/plain" });
+//   res.end(version);
+// };
```
Comment (theMr17) [inline src/db/database.js:14]: Remove the commented code.

File: .env.sample
Comment (theMr17) [inline .env.sample:1]: Add a new line at EOF. Better to update the `.prettierrc` file with the `"insertFinalNewline": true` field to automatically add a new line everytime.

File: src/controllers/healthcheck.controller.js
Comment (theMr17) [inline src/controllers/healthcheck.controller.js:1]: Update the file name to `healthCheck.controller.js`. Do the same for `healthCheck.routes.js`.

Comment (rahulc0dy) [reply src/controllers/healthcheck.controller.js:1]: ok

**Review by theMr17 — APPROVED** [review]

**Pull request merged**